### ISSUE TITLE
Getting stuck after "Loading the model data" with ROS Kinetic

### ISSUE THREAD

**Issue by Allopart** [kunle12/dn_object_detect#5]
I am trying to implement dn_object_detect on ROS Kinetic (Ubuntu 16.04, CUDA 8.0, OpenCV 2.4.13) but I keep running into the problem that it gets stuck after :

 _Loaded detection model data_

The .cfg and .weight files are loaded correctly and the camera topic publishes values too.
In the past I have successfully implemented it in ROS Indigo (Ubuntu 14.04, CUDA 7.5) and that is why I wanted to move further with it.
Any suggestions?
Kind regards

**Comment by kunle12** [comment]
OK. It just happens that I will in fact test the code on this exact set up tomorrow (don't tell me that you are trying to run dn_object_detect on a TX2 board ;)). I will let you know how I go.

**Comment by Allopart** [comment]
Great! Looking forward to hearing from you.

**Comment by Allopart** [comment]
Figured out the problem: for whatever reason Image_transport segfaults if using a compressed image.
To solve it:
1. In the .launch file, change "compressed" tag for "raw"
2. In src/MultiClassObjectDetector.cpp, in line image_transport::TransportHints hints( "compressed" ) change to "raw"

**Comment by kunle12** [comment]
Yes. This is what I have also encountered. There must be some bugs in the image transport code which is beyond the scope of this project. However, I have another code that is very similar to dn_object_detect, it works perfectly under the same environment. I'm struggling to find out what would possibly trigger this problem.
